Algorithm-Binary-Agents.md

```diff
@@ -1,6 +1,8 @@
+# Algorithm Binary Agents
+
 ![](http://i.imgur.com/HSwaSFK.jpg)
 
-# Explanation:
+## Explanation:

```

Comment:
The Explanation should appear in chat at the first call. With `h2`, it won't appear.